pkg/apis/core/v1beta1/types_shoot.go

```diff
@@ -723,6 +723,10 @@ type KubeAPIServerConfig struct {
 	// Defaults to 300.
 	// +optional
 	DefaultUnreachableTolerationSeconds *int64 `json:"defaultUnreachableTolerationSeconds,omitempty" protobuf:"varint,15,opt,name=defaultUnreachableTolerationSeconds"`
+
+	//CORSAllowedOrigins contains a list of allowed origins for CORS, comma separated. An allowed origin can be a regular expression to support subdomain matching. If this list is empty CORS will not be enabled. (flag `--cors-allowed-origins`)
+	// +optional
+	CORSAllowedOrigins []string `json:"corsAllowedOrigins,omitempty" protobuf:"bytes,16,rep,name=corsAllowedOrigins"`
```

Comment:
Does it make sense to add some validation for this field, e.g., to ensure only valid domains are entered (maybe other restrictions, if needed)?